### ISSUE TITLE
Val_rpn_loss is increasing 

### ISSUE THREAD

**Issue by vyadava** [matterport/Mask_RCNN#2475]
I am training a dataset of 580 images and validating the model on 50 images.  Earlier i ran  the training i found all losses were decreasing but dont know if i have changed any config setting or something else. Now I am seeing a weird behaviour in Val_rpn_class_loss and Val_rpn_bbox_loss. its continuosly increasing causing total Val_loss to increase. Have anybody came across similar situation? Please help with your suggestion

![image](https://user-images.githubusercontent.com/50803095/107006463-27d6d980-67b7-11eb-8f9c-6e226f4e0194.png)

**Comment by MonchoG** [comment]
Hello @vyadava . I have been using this implementation of Mask RCNN in my project as well. I observed the same behaviour during my training sessions as well. 
My dataset consists of 262 images, 60 of which are in the validation split. My training metrics show decent results, decreasing as training goes, but my validation metrics are fluctuating a lot and the end validation metrics are higher compared to the initial values.
My suspicion is that the dataset is small, especially the validation split, but  I can't tell for sure since I am not an expert in the topic. Did you manage to solve your problem or did you get any other insights ?

**Comment by vyadava** [comment]
In my case, tuning learning rate helped in converging the validation loss.

**Comment by georgebakas** [comment]
How much did you change the learning rate? I tried switching to a 0.01 but got NaN values in Loss...
